MbDotNet/Models/Imposters/RetrievedSmtpImposter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using MbDotNet.Models.Requests;
+using MbDotNet.Models.Responses.Fields;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace MbDotNet.Models.Imposters
+{
+    public class RetrievedSmtpImposter: RetrievedImposter<SmtpRequest, HttpResponseFields>
```

Comment:
Instead of using `HttpResponseFields` for the second type here, I would create an empty `SmtpResponseFields`.

Reply:
Done